watch.d/SRS_ALTIMETRY.json

```diff
@@ -1,5 +1,5 @@
 {
   "path": [ "SRS/ALTIMETRY" ],
   "execute": "bin/generic_incoming_handler.sh",
-  "execute_params": "--exec SRS/SRS_ALT/dest_path.py --regex '^IMOS_SRSALT_[[:alpha:]]{2,3}_[[:digit:]]{8}T[[:digit:]]{6}Z_.?{7}_FV01_(SBE26|Aquad|SBE37)d[[:digit:]]*m\\.nc$' --checks 'cf imos:1.3'"
+  "execute_params": "--exec SRS/SRS_ALT/dest_path.py --regex '^IMOS_SRSALT_[[:alpha:]]*_[[:digit:]]{8}T[[:digit:]]{6}Z_.?{7}_FV01_(SBE26|Aquad|SBE37)d[[:digit:]]*m\\.nc$' --checks 'cf imos:1.3'"
```

Comment:
I cannot find the document anymore, it was in an old NCRIS funding plan. But the Facility code was "SRSALT". I guess SRS-ALT would be better